components/tokens/index.js

```diff
@@ -78,7 +78,16 @@ const getVotingPowerAtLevel = async (req, response) => {
       userAddress
```

Comment:
here too xtz = false would be easy to understand

Reply:
added!